src/plugins/showHiddenThings/index.ts

```diff
@@ -80,11 +80,19 @@ export default definePlugin({
             }
         },
         {
-            find: "auto_removed:",
+            find: "prod_discoverable_guilds",
             predicate: () => settings.store.disableDiscoveryFilters,
             replacement: {
-                match: /filters:\i\.join\(" AND "\),facets:\[/,
-                replace: "facets:["
+                match: /\{"auto_removed:.*?\}/,
+                replace: "{}"
+            }
+        },
+        {
+            find: "\">200\"",
```

Comment:
Patch the getter for MINIMUM_MEMBER_COUNT instead. I know in the past I said it's bad to patch exports, but in this case it's better than targetting a nunber

```suggestion
            find: "MINIMUM_MEMBER_COUNT:",
```

Reply:
Done!

Reply:
clueless